modelskill/plot.py

```diff
@@ -443,6 +453,9 @@ def scatter(
         show the data density as a colormap of the scatter, by default None. If both `show_density` and `show_hist`
         are None, then `show_density` is used by default.
         for binning the data, the previous kword `bins=Float` is used
+    norm : matplotlib.colors norm
```

Comment:
Can a norm also be given as a str like directly in matplotlib, see e.g. "norm" in https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.contourf.html ?

Reply:
I tried to implement that now, but (in my version of mpl - 3.5.3) the `matplotlib.scale` and `matplotlib.colors.Normalize` classes are incompatible.

I will need to update my matplotlib or manually add the `str` functionality. Will add as a todo.

Reply:
Don't worry - we can always do that some other time :-)